src/ontology/imports/foodon_product_import.owl

```diff
@@ -33599,6 +33605,109 @@ Fishcakes as defined in the Oxford Dictionary of Food and Nutrition are chopped
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00004166">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_00002574"/>
+        <obo:IAO_0000117 rdf:resource="https://orcid.org/0000-0001-5275-8866"/>
+        <dc:date rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2022-01-04T21:24:34Z</dc:date>
+        <rdfs:label xml:lang="en">cut of beef, mexican</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/FOODON_00004167 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00004167">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_00004166"/>
+        <obo:IAO_0000114 rdf:resource="http://purl.obolibrary.org/obo/IAO_0000428"/>
+        <obo:IAO_0000115 xml:lang="en">A cut of beef and part of the sub-prime cut known as the chuck.</obo:IAO_0000115>
+        <obo:IAO_0000117 rdf:resource="https://orcid.org/0000-0001-5275-8866"/>
+        <obo:IAO_0000119>https://rollybrook.com/lexicon-groups.htm</obo:IAO_0000119>
+        <obo:IAO_0000119>https://www.mexconnect.com/articles/2398-choice-cut-or-mystery-meat-a-guide-to-mexican-butcher-shops-part-i-beef/</obo:IAO_0000119>
+        <rdfs:label xml:lang="en">diezmillo</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/FOODON_00004168 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00004168">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_03317170"/>
+        <obo:IAO_0000115 xml:lang="en">The process of slow cooking meat to make it easier to chew or cut.</obo:IAO_0000115>
+        <obo:IAO_0000117 rdf:resource="https://orcid.org/0000-0001-5275-8866"/>
+        <obo:IAO_0000119 xml:lang="en">https://en.wikipedia.org/wiki/Meat_tenderness</obo:IAO_0000119>
+        <dc:date rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2022-02-04T22:11:53Z</dc:date>
+        <rdfs:label xml:lang="en">meat (tenderized)</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/FOODON_00004169 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00004169">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_00001180"/>
+        <obo:IAO_0000115 xml:lang="en">The chicken wings which have been prepared.</obo:IAO_0000115>
+        <obo:IAO_0000117 rdf:resource="https://orcid.org/0000-0001-5275-8866"/>
+        <dc:date rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2022-01-04T22:15:34Z</dc:date>
+        <rdfs:label xml:lang="en">party wings</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/FOODON_00004170 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00004170">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_03309997"/>
+        <obo:IAO_0000115 xml:lang="en">Any foodstuff used to feed livestock.</obo:IAO_0000115>
```

Comment:
I can't quite recall our discussion about this, but there was the distinction between "feed" like foraging crops as meant for mainly herbaceous creatures; whereas a more general term for any kind of feedstuff for animals was needed.  But FoodOn already has ENVO term "animal feed".  Should we ask ENVO to take over this term, and add the exact synonyms? Or ask them to add the synonyms?

Reply:
I think in the very end we discussed to organize it somewhat like this (which I have in my notes). 
--feed stuff
---animal feed (ENVO term?)
-----fodder (animal feed)
FoodOn has 'foodstuff' as a synonym of 'food material'. 
And definition of 'animal feed' says that '`Food material` which can be used to meet nutritional requirements of animals, particularly livestock such as cattle, goats, sheep, horses, chickens and pigs.
so does it makes sense to have something like this? 
-food material
--feed stuff
---animal feed (ENVO term?)
-----fodder (animal feed)

Reply:
I remember that you also proposed a hierarchy in zoom chat (14Dec), but unfortunately Zoom didn't save my chat. Do you still have that?

Reply:
I think ENVO "animal feed" is the broadest term under which all food materials intended for animals is placed. Wikipedia backs this up: https://en.wikipedia.org/wiki/Animal_feed   
Given FoodOn's mandate, I think "animal feed" could be converted to a new FOODON term.  Can you put in a request for that to ENVO to see what they say?  Then we can have that term under "food product for animal".  And then add feedstuff and feed stuff as exact synonyms.  I'm seeing this now from the diagram in http://www.ndvsu.org/images/StudyMaterials/Nutrition/Classification-of-feedstuffs.pdf 

Then FoodOn "fodder (animal feed)" can be placed under "animal feed".

Reply:
Just to confirm, from 'food for animals', we mean 'food product for animals', right? 
 like this:
-food product for animals
--feedstuff (food for agricultural animals)
    fodder (animal feed)
--pet food
--cat food
--dog food

Reply:
Right!

Reply:
p.s. cat food and dog food would be indented under pet food (as they are now).

Reply:
Exactly!